app/static/src/components/Products.vue

```diff
@@ -1,24 +1,14 @@
 <template>
   <div class="card mb-3">
     <div class="card-header">
-      <a href="#" @click.prevent="show = !show" id="toggle-products">
-        Add Products  <span class="oi" :class="show ? 'oi-minus' : 'oi-plus'"></span>
+      <a href="#" @click.prevent="show = !show" id="add-products-btn">Add-Products<span class="oi" :class="show ? 'oi-minus' : 'oi-plus'"></span>
       </a>
     </div>
     <div class="card-body" v-show="show">
       <div class="form-group row" v-for="product in products">
         <label :for="`product-${product.id}`" class="col-sm col-form-label col-form-label-sm">{{ product.name }}</label>
         <div class="col-sm">
-          <select class="form-control form-control-sm"
-            :id="`product-${product.id}-select`"
-            v-model="value[product.id]"
-            @change="updateProducts">
-            <option>1</option>
-            <option>2</option>
-            <option>3</option>
-            <option>4</option>
-            <option>5</option>
-          </select>
+          <input type="number" class="form-control" :id="`product-${product.id}-quantity`" v-model="value[product.id]" placeholder="0" @change="updateProducts"/>
```

Comment:
Just curious why `input` over `select`?

Reply:
It just made more sense to be able to add any number instead of being limited to just 5 and that also has an id now.